web/styles/utils.sass

```diff
@@ -0,0 +1,62 @@
+.fill
+  &-width
+    width: 100%
+
+  &-height
+    height: 100%
+
+.split
+  > *
+    width: 50%
+
+// padding declarations
+.pad
+  &-sides
```

Comment:
I think we might want to be more specific here like `pad-sides-xl` so that we can vary the size of padding, but we can that on an ongoing basis

Reply:
Yeah, if we see the need for more we should add them on an ongoing basis - not sure if this is `xl` or not quite yet 😋